org.nongnu.enigma.yml

```
@@ -0,0 +1,88 @@
+app-id: org.nongnu.enigma
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+appstream-compose: false
+command: enigma
+
+finish-args:
+  - --persist=.enigma
+  # X11 access
+  - --share=ipc
+  - --socket=x11
+  # Needs to talk to pulseaudio
+  - --socket=pulseaudio
+  # OpenGL redering
+  - --device=dri
+
+modules:
+  - name: texi2html
+    sources:
+      - type: archive
+        url: https://download.savannah.gnu.org/releases/texi2html/texi2html-5.0.tar.gz
+        sha256: e60edd2a9b8399ca615c6e81e06fa61946ba2f2406c76cd63eb829c91d3a3d7d
```

```suggestion
        sha256: e60edd2a9b8399ca615c6e81e06fa61946ba2f2406c76cd63eb829c91d3a3d7d
      - type: shell
        commands:
          - cp -p /usr/share/automake-*/config.{sub,guess} .
```

You need to add this for building on other arch.